server/server.go

```diff
@@ -218,17 +218,29 @@ func New(c *Config) (*Server, error) {
 		Base: basePolicyHandler,
 	}
 
+	oauth2Params := []oauth2.Param{
+		oauth2.ForceTLS(forceTLS),
```

Comment:
The `ForceTLS` option is only used to generate URLs, so we should be able to remove it now that we're setting a fixed URL.